lib/vendor/inherits.js

```diff
@@ -0,0 +1,24 @@
+module.exports = function inherits(ctor, superCtor) {
```

Comment:
Please add a comment at the top of this file explaining why we are pulling in this. Also add the license header, grab it from the author https://github.com/isaacs/inherits/blob/main/LICENSE. See vendor/loglevel.js for reference.